### PR TITLE
Adding links to Programs page - related #30

### PR DESCRIPTION
**Pull request by gvn**
Adds the links that are currently available...

### CODE REVIEW

File: source/pug/views/programs.pug [hunk 1]
```diff
@@ -37,17 +37,17 @@ block content
...
 
     .row.mb-5
       .col-md-6
         h2.h3-black Grants
         p We provide funding to promising people and big ideas. Our grants support gigabit technology laboratories, critical studies in emerging markets and more.
         ul
-          li.mb-3: a.cta-link(href=``) Apply for Gigabit grant 2017
-          li.mb-3: a.cta-link(href=``) See grant project examples (NYC/Chicago)
+          li.mb-3: a.cta-link(href=`https://learning.mozilla.org/en-US/gigabit/apply`) Apply for Gigabit grant 2017
+          li.mb-3: a.cta-link(href=`http://hivechicago.org/portfolio/#/*`) See grant project examples (NYC/Chicago)
```
Comment (alanmoo) [inline source/pug/views/programs.pug]: `*` intentional?

Comment (gvn) [inline source/pug/views/programs.pug]: Apparently

File: source/pug/views/programs.pug [hunk 2]
```diff
@@ -58,17 +58,17 @@ block content
         h2.h3-black Local Organizing
         p We connect passionate people in cities across five continents, from technologists in Chattanooga to educators in Cape Town. Our Network provides organizing best-practices and mutual support. In certain cities, our Network can also deploy on-the-ground staff.
         ul
-          li.mb-3: a.cta-link(href=``) Mozilla Gigabit cities
-          li.mb-3: a.cta-link(href=``) Mozilla clubs
-          li.mb-3: a.cta-link(href=``) Hive learning cities
+          li.mb-3: a.cta-link(href=`https://learning.mozilla.org/gigabit`) Mozilla Gigabit cities
+          li.mb-3: a.cta-link(href=`https://learning.mozilla.org/en-US/clubs`) Mozilla clubs
+          li.mb-3: a.cta-link(href=`https://hivelearningnetworks.org/`) Hive Learning Networks
```
Comment (alanmoo) [inline source/pug/views/programs.pug]: Error establishing a database connection

File: source/pug/views/programs.pug [hunk 3]
```diff
@@ -58,17 +58,17 @@ block content
...
 
     .row.mb-5
       .col-md-6
         h2.h3-black Research
         p Building a healthy Internet requires data and insight: We need to know what deserves our attention. Network research investigating issues like the global app economy, digital skills in classrooms and smartphone use in emerging markets
         ul
-          li.mb-3: a.cta-link(href=``) See our global research reports
-          li.mb-3: a.cta-link(href=``) See the Internet Health Report
+          li.mb-3: a.cta-link(href=`https://github.com/mozilla/network/issues/mzl.la/globalweb`) See our global research reports
```
Comment (alanmoo) [inline source/pug/views/programs.pug]: 404

**Pull request merged**